src/azure-cli/azure/cli/command_modules/acr/task.py

```diff
@@ -34,6 +37,7 @@
 ALLOWED_TASK_FILE_TYPES = ('.yaml', '.yml', '.toml', '.json', '.sh', '.bash', '.zsh', '.ps1',
                            '.ps', '.cmd', '.bat', '.ts', '.js', '.php', '.py', '.rb', '.lua')
 
+NULL_SOURCE_LOCATION = "/dev/null"
```

Comment:
Please ignore the redundant imports. I was testing some code and forgot to remove it. I will send a new iteration removing it thanks!

Reply:
could you move this value to the _constants.py please?